packages/analytics-ma-browser/scripts/templates/browser-snippet.template.js

```diff
@@ -0,0 +1,123 @@
+const snippet = (integrity, version) => `
```

Comment:
i feel like this template should in common and share with browser so we only update it once. easy to miss updating both

Reply:
Right. Will do it in a separate PR.